### PR TITLE
WEB-1117-add2fa

### PR DESCRIPTION
**Pull request by ginnyyadav**
Tidied some syntax up. Fixed a loading issue timeout with ther HelpWidget. Added OTP library to access 2FA.

Edit: the first half of the commits are pretty straightforward. the last half was me fixing ci failures.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,13 +21,14 @@
   "homepage": "https://github.com/tidepool-org/nightwatchtests#readme",
   "dependencies": {
     "dayjs": "1.9.1",
+    "dotenv": "8.2.0",
     "eslint-config-airbnb": "18.2.0",
-    "nightwatch": "1.4.2"
+    "nightwatch": "1.4.2",
+    "otplib": "^12.0.1"
```
Comment (cbwebdevelopment) [inline package.json:27]: I'd recommend locking down package versions
```suggestion
    "otplib": "12.0.1"
```

File: package.json [hunk 2]
```diff
@@ -21,13 +21,14 @@
...
   },
   "devDependencies": {
-    "dotenv": "8.2.0",
-    "eslint": "7.10.0",
-    "eslint-config-airbnb-base": "14.2.0",
-    "eslint-plugin-import": "2.22.1"
+    "eslint": "7.18.0",
+    "eslint-config-airbnb-base": "14.2.1",
+    "eslint-plugin-import": "^2.22.1"
```
Comment (cbwebdevelopment) [inline package.json:32]: I'd recommend locking down package versions
```suggestion
    "eslint-plugin-import": "2.22.1"
```

File: pageobjects/ForgotPasswordPage.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
+const { getToken } = require('../utilities/utilities');
```
Comment (cbwebdevelopment) [inline pageobjects/ForgotPasswordPage.js:1]: Curious why there are now 2 identical page objects - `forgotPasswordPage` and `ForgotPasswordPage`?

I'm guessing that this one was accidentally committed.

**Review by cbwebdevelopment — CHANGES_REQUESTED** [review]
Looks good.  I just a couple small questions / requested changes.

File: package.json (hunk 1 above)
Comment (ginnyyadav) [reply package.json:27]: oh gosh i missed this!. thanks!

File: pageobjects/ForgotPasswordPage.js (hunk 3 above)
Comment (ginnyyadav) [reply pageobjects/ForgotPasswordPage.js:1]: oh dear....will fix that

**Review by cbwebdevelopment — APPROVED** [review]
LGTM 🚢

**Pull request merged**